.github/workflows/setup-gcloud-it.yml

```diff
@@ -17,7 +17,7 @@ jobs:
   versioned:
     if: ${{ github.event_name == 'push' || github.repository == github.event.pull_request.head.repo.full_name && github.actor != 'dependabot[bot]' }}
     name: setup-gcloud versioned
-    runs-on: ${{ matrix.operating-system }}
+    runs-on: ${{ matrix.os }}
```

Comment:
Can you move this below the matrix everywhere?

Reply:
updated